khal/cli.py

```diff
@@ -93,6 +93,12 @@ def multi_calendar_option(f):
     return d(a(f))
 
 
+def no_mouse_option(f):
```

Comment:
Since I had to use this both for `ikhal` and `khal interactive`, I figured it made sense to abstract this option into its own function decorator.